roles/keystone/tasks/keystone-idp.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: setup keystone idp metadata
+  shell: keystone-manage saml_idp_metadata > /etc/keystone/keystone_idp_metadata.xml
+  when: keystone.keystone_identity_provider.enabled|bool
+  tags: k2k-federation
```

Comment:
This will need to change if keystone.conf has been changed in the `[saml]` section. I will look to see if there is any easy way of checking this.